File: 백준/Bronze/10101. 삼각형 외우기/삼각형 외우기.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args) {
       Scanner scan = new Scanner(System.in);
       
       int A = scan.nextInt();
       int B = scan.nextInt();
       int C = scan.nextInt();
       
       if(A+B+C != 180) {
    	   System.out.println("Error");
       }else if((A==60)&&(C==60)&&(B==60)) {
    	   System.out.println("Equilateral");
       }else if((A+B+C == 180)&&(A!=B)&&(A!=C)&&(B!=C)){
    	   System.out.println("Scalene");
       }else if((A+B+C == 180)&&(A==B)||(A==C)||(B==C)){
    	   System.out.println("Isosceles");
       }
    } 
}

File: 백준/Bronze/10101. 삼각형 외우기/삼각형 외우기.py
A = int(input())
B = int(input())
C = int(input())

if A + B + C != 180:
    print("Error")
elif A == 60 and B == 60 and C == 60:
    print("Equilateral")
elif A != B and A != C and B != C:
    print("Scalene")
else:
    print("Isosceles")